cmd/monaco/download/download_configs.go

```diff
@@ -124,6 +125,11 @@ func (d DefaultCommand) DownloadConfigsBasedOnManifest(fs afero.Fs, cmdOptions d
 		onlyAutomation:  cmdOptions.onlyAutomation,
 	}
 
+	if errs := options.valid(); len(errs) != 0 {
+		err := printAndFormatErrors(errs, "input data are not valid")
```

Comment:
```suggestion
		err := printAndFormatErrors(errs, "command options are not valid")
```

Input data sounds pretty vague